docs/modules/ROOT/pages/jet-engine-configuration.adoc

```diff
@@ -60,6 +60,12 @@ The Jet engine is enabled by default and has the following configuration options
   - `secret`: Name of the Secret object which holds the credentials for your cloud provider.
   - `bucketURI`: Full path for the external bucket. For example: `gs://your-bucket/path/to/jars`.
 
+|configMaps
+|List of names of ConfigMaps. Files in each ConfigMap will be downloaded.
+
+|remoteURLs
+|List of URLs from where the files will be downloaded.
+
```

Comment:
You skipped to add Default column in new rows. That is why the new rows are placed in the wrong cells.